osm2gtfs/creators/incofer/trips_creator_incofer.py

```diff
@@ -11,20 +11,22 @@ class TripsCreatorIncofer(TripsCreator):
 
     def add_trips_to_feed(self, feed, data):
 
-        lines = data.routes
+        lines = data.get_routes()
 
         # line (osm rounte master | gtfs route)
         for line_id, line in lines.iteritems():
             # debug
             # print("DEBUG. procesando la línea:", line.name)
 
             # itinerary (osm route | non existent gtfs element)
-            for itinerary_id, itinerary in line.routes.iteritems():
+            itineraries = line.get_itineraries()
+            for itinerary in itineraries:
                 # debug
                 # print("DEBUG. procesando el itinerario", itinerary.name)
 
                 # shape for itinerary
-                shape_id = TripsCreator.add_shape(feed, itinerary_id, itinerary)
+                shape_id = self._add_shape_to_feed(
+                    feed, itinerary.route_id, itinerary)
```

Comment:
When invoking  `_add_shape_to_feed` with `itinerary.route_id` I get a bug with the shapes. See mapanica/osm2gtfs#1